mathsnuggets/widgets/quadratic_equations.py

```diff
@@ -0,0 +1,21 @@
+import sympy
+
+from mathsnuggets.core import fields
+from mathsnuggets.widgets import equation
+
+test = {"equation": "x^2 - 3x + 2"}
+
+
+class QuadraticEquations(equation.Equation):
+    """Quadratic Equations"""
+
+    a = fields.RandomNumber("a")
+    b = fields.RandomNumber("b")
+    c = fields.RandomNumber("c")
+
+    def generator(self):
+        self.equation = sympy.Eq(self.a * self.x ** 2 + self.b * self.x + self.c)
```

Comment:
Mathematically, not sure it's the best form to work with... For example, writing a constraint that the solutions should be integers is needlessly long